enterprise/migrations/0026_remove_old_consent.py

```diff
@@ -1,48 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-        ('enterprise', '0025_auto_20170828_1412'),
-    ]
-
-    operations = [
-        migrations.RemoveField(
-            model_name='historicaluserdatasharingconsentaudit',
-            name='history_user',
-        ),
-        migrations.RemoveField(
-            model_name='historicaluserdatasharingconsentaudit',
-            name='user',
-        ),
-        migrations.RemoveField(
-            model_name='userdatasharingconsentaudit',
-            name='user',
-        ),
-        migrations.RemoveField(
-            model_name='enterprisecourseenrollment',
-            name='consent_granted',
-        ),
-        migrations.RemoveField(
-            model_name='enterprisecustomer',
-            name='require_account_level_consent',
-        ),
```

Comment:
I think before we entirely remove this field from code, we must first mark it as nullable and deploy the migration associated with that change. We can then remove the field entirely from code and deploy, then deploy the migration to remove it from the database.

Reply:
That sounds pretty complex, and I'm starting to wonder if this is necessary for work outside of the platform (I wouldn't imagine we could *ever* do this for external dependencies that made such a migration, for example, but things work out fine). The main reason I'm doubtful is that, in order for this migration to come up *in the first place*, we have to have installed the latest edx-enterprise version, which comes with the latest code. I could still be wrong so maybe we should double-check with ops?

Reply:
It is complex, but it is necessary. I imagine that if we have done something similar in the past that the feature which touched the models with removed fields was broken for the time between when we run the migrations and when the servers with the new code start servicing users.

Reply:
Alright, so I'll add back the fields in this PR and make them nullable, and generate the migration for that and we can deploy it. In #197 I'll remove these fields only but not make any new migration. In a new PR we'll finally make the migration. Jeez :P